main.py

```diff
@@ -1,10 +1,49 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
```

Comment:
```suggestion
    with open("players.json") as file:
```
`r` mode set by deffault